composer.json

```
@@ -28,7 +28,7 @@
     "php-coveralls/php-coveralls": "^2.4",
     "php-mock/php-mock-phpunit": "^1.1 || ^2.0",
     "php-parallel-lint/php-parallel-lint": "^1.2",
-    "phpunit/phpunit": "^5.7 || ^7 || ^8 || ^9",
+    "phpunit/phpunit": "^5.7,>=5.7.15 || ^7 || ^8 || ^9",
```

```suggestion
    "phpunit/phpunit": "^5.7,>=5.7.15 || ^7 || ^8 || ^9",
```
Could you explain why this is need please ?
it looks wrong because `^5.7` would include `>=5.7.15`

---

phpunit < 5.7.15 has a bug with yield.

---

Verrrrry late reply, you can simply use `^5.7.15`, which is perhaps less ambiguous than the current one.